src/screens/Sidebar/Sidebar.res

```diff
@@ -553,51 +555,75 @@ let make = (
+                    key={Int.toString(index)}
+                    section
+                    linkSelectionCheck
+                    firstPart
+                    isSideBarExpanded={isSidebarExpanded}
+                    openItem
+                    setOpenItem
+                    isSectionAutoCollapseEnabled=true
+                  />
+                </RenderIf>
+              | Heading(headingOptions) =>
+                <div
                   key={Int.toString(index)}
-                  section
-                  linkSelectionCheck
-                  firstPart
-                  isSideBarExpanded={isSidebarExpanded}
-                  openItem
-                  setOpenItem
-                  isSectionAutoCollapseEnabled=true
-                />
-              </RenderIf>
-            | Heading(headingOptions) =>
-              <div
-                key={Int.toString(index)}
-                className={`text-xs font-semibold leading-5 text-[#5B6376] overflow-hidden border-l-2 rounded-sm border-transparent px-3 ${isSidebarExpanded
-                    ? "mx-2"
-                    : "mx-1"} mt-5 mb-3`}>
-                {{isSidebarExpanded ? headingOptions.name : ""}->React.string}
-              </div>
+                  className={`text-xs font-semibold leading-5 text-[#5B6376] overflow-hidden border-l-2 rounded-sm border-transparent px-3 ${isSidebarExpanded
+                      ? "mx-2"
+                      : "mx-1"} mt-5 mb-3`}>
+                  {{isSidebarExpanded ? headingOptions.name : ""}->React.string}
+                </div>
 
-            | CustomComponent(customComponentOptions) =>
-              <RenderIf condition={isSidebarExpanded} key={Int.toString(index)}>
-                customComponentOptions.component
-              </RenderIf>
-            }
-          })
-          ->React.array}
+              | CustomComponent(customComponentOptions) =>
+                <RenderIf condition={isSidebarExpanded} key={Int.toString(index)}>
+                  customComponentOptions.component
+                </RenderIf>
+              }
+            })
+            ->React.array}
+          </div>
+          <div className="border-t border-blue-840 mt-5">
+            {productSiebars
+            ->Array.mapWithIndex((tabInfo, index) => {
+              switch tabInfo {
+              | Section(section) =>
+                <RenderIf condition={section.showSection} key={Int.toString(index)}>
+                  <SidebarNestedSection
+                    key={Int.toString(index)}
+                    section
+                    linkSelectionCheck
+                    firstPart
+                    isSideBarExpanded={isSidebarExpanded}
+                    openItem
+                    setOpenItem
+                    isSectionAutoCollapseEnabled=true
+                  />
+                </RenderIf>
+              | _ => React.null
+              }
+            })
+            ->React.array}
+          </div>
```

Comment:
Even if we provide empty productSidebars it will render top border , can we wrap it with RenderIf and condition productSidebars.length>0